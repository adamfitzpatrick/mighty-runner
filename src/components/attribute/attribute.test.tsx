import * as React from 'react'
import * as renderer from 'react-test-renderer'
import Attribute from '.'

describe('Attribute component', () => {
  test('should do nothing', () => {
    expect(true).toEqual(true)
  })
})
