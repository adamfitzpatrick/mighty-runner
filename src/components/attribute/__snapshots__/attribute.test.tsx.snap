// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Attribute component display component should render correctly 1`] = `"<div class=\\"attribute\\"><div><h3>Agility (agi)</h3><div><span>Effective Value:<span data-testid=\\"attributes.agility.effective-stat.container\\">3</span></span><button data-testid=\\"Edit.button.component\\">Edit</button></div><div data-testid=\\"non-editable-effect.component\\"><span>Name: n</span><label><span>Active</span><input data-testid=\\"Active.input.component\\" type=\\"checkbox\\" value=\\"\\" checked=\\"\\"></label></div></div></div>"`;

exports[`Attribute component edit component should render correctly 1`] = `"<div><h3>Agility</h3><label><span>Maximum</span><input data-testid=\\"Maximum.input.component\\" type=\\"text\\" value=\\"6\\"></label><label><span>Initial Value</span><input data-testid=\\"Initial Value.input.component\\" type=\\"text\\" value=\\"1\\"></label><label><span>Chargen</span><input data-testid=\\"Chargen.input.component\\" type=\\"text\\" value=\\"1\\"></label></div>"`;
